robotsim.py

```diff
@@ -553,7 +476,9 @@ def main():
             reset = False
         else:
             run_button = pygame.transform.scale(run_button, (int(pixel_constant*0.5), int(pixel_constant*0.5)))
+            rosa_de_los_vientos = pygame.transform.scale(rosa_de_los_vientos, (int(pixel_constant), int(pixel_constant)))
             gameDisplay.blit(run_button, (0, 0))
+            gameDisplay.blit(rosa_de_los_vientos, (0, pixel_constant*8))
```

Comment:
Is rosa_de_los_vientos necessary here? I believe it would be enough with it in generate_map().

Reply:
this actually resolved the issue that made the image pixelated, thank you!